CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Thank You
+
+We are grateful that you considered contributing to metaKGP-wiki. Please follow the below guidelines for contribution.
+
+Follow the golder rule
+
+> Code, Document, Raise Issues and Review like you have to explain everything to a new developer without saying a word.
+
+## First Time
+
+The first thing to do always is to run the program and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches in the sequential manner.
+
+Also take a note of the coding style and dependencies that the project follows.
+
+## Choosing issues
+
+It is recommended to go through the issues tab once you have ran the program once and look for easy issues.
+
+If an issue is not assigned; follow through the comments, ask queries about it and then start working on it.
+
+## Creating issues
+
+If you face any reproducable problem while launching the game or any feature you think should be there, file an issue in the github repository.
+
+It is expected that you will follow the below guidelines while creating issues:
+
+https://wiredcraft.com/blog/how-we-write-our-github-issues
+
+## Sending a PR
+
+While sending a PR, always remember not to send one from your master branch; it'll lead to commit issues and mismatch. 
+
+It is highly recommended to follow the below guidelines while writing commits and sending PRs:
+
+- https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/
+- https://code.likeagirl.io/useful-tips-for-writing-better-git-commit-messages-808770609503
+
+## Communication
+
+Please join the [gitter channel](https://gitter.im/metakgp-wiki/Lobby) for communication regarding the project. No personal communication will be entertained :)
```

Comment:
Not sure who is following gitter. I think we should redirect them to slack from here.